modules/cnf-image-based-upgrade-with-backup.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-image-based-upgrade-base.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrading-with-backup_{context}"]
+= Moving to the Upgrade stage of the image-based upgrade with {lcao}
```

Comment:
It is the name of the stage, so like `Upgrade` stage but I can't use backticks in titles

Reply:
ACK - in that case does [line 21](https://github.com/openshift/openshift-docs/pull/77477/files#diff-be40bc678801dde183d4574e926410e4aa50d681b5e2ca87b9bbc1a3498f0e90R21) need `Upgrade` instead of upgrade?

Reply:
That's correct, I'll update it in a sec!